src/main/java/gyro/google/kms/CryptoKeyFinder.java

```diff
@@ -0,0 +1,73 @@
+package gyro.google.kms;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.google.cloud.kms.v1.CryptoKey;
+import com.google.cloud.kms.v1.KeyManagementServiceClient;
+import com.google.cloud.kms.v1.KeyRingName;
+import gyro.core.GyroException;
+import gyro.core.Type;
+import gyro.google.GoogleFinder;
+
+/**
+ * Query for crypto keys.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    crypto-key: $(external-query google::crypto-key {location: "us-east4", key-ring-name: "key-ring-example"})
+ */
+@Type("crypto-key")
+public class CryptoKeyFinder extends GoogleFinder<KeyManagementServiceClient, CryptoKey, CryptoKeyResource> {
+    private String location;
+    private String keyRingName;
```

Comment:
rename variable and corresponding use cases and example
```suggestion
    private String name;
```